ui/docs/manual/design/apis/hawk/README.md

```diff
@@ -7,7 +7,7 @@ order: 10
 All Taskcluster components use [Hawk](https://github.com/hueniverse/hawk) over
 SSL for authentication and authorize access based on a set of "scopes"
 associated with each client. Credentials and scopes are managed by the
-[authentication](/docs/reference/platform/taskcluster-auth/references/api) component.
```

Comment:
I think this transform is the biggest cause of 404s (for my usage anyway).  Maybe we could encode some route translation to do this automatically?

Reply:
Sure. I think we can do something like:

```jsx
const path = require('path');
const re = /(\/docs\/reference\/.*)\/(?:taskcluster-)(.*)(?:\/references)\/(api)/;
const matches = request.uri.match(re);

if (matches) {
  const uri = path.join(matches[1], matches[2], matches[3]);
  request.uri = uri;
  return callback(// ...);
}

// continue with the rest
```

The AWS console seems to be down. I'll do it once it's back up.